src/components/Grunnlag/sections/Pensjonsavtaler.module.scss

```diff
@@ -43,7 +43,21 @@
   font-size: var(--a-font-size-medium);
 }
 
+.error {
+  display: flex;
+
+  &Icon {
+    flex-basis: 1.5rem;
+    color: var(--a-orange-600);
+    vertical-align: top;
+    margin-right: var(--a-spacing-2);
+  }
+
+  &Text {
+    flex-basis: 100%;
+  }
+}
+
```

Comment:
Er i grunn ikke tilhenger av å måtte manuelt konkatenere strings i hodet for å se hvilke klassenavn det er snakk om her. Når man ser `&Text` må man finne klassenavnet i blokken over, `.error`, og så slå de sammen for å se hvilken klasse det er. 

```css
.error {
   display: flex
}

.errorText {
  flex-basis: 100%
}
```

er mer lesbart enn

```css
.error {
  display: flex;

  ...

  &Text {
    flex-basis: 100%;
  }
}
```

Det blir virkelig ille når man går dypere enn 2 nivåer, noe vi gjør enkelte steder i kodebasen

Reply:
Dette mener jeg vi har diskutert før. Det er standard scss kode og tipper at det er en vanesak.
Jeg tror som sagt ikke at det er helt riktig at jeg ikke skal kunne bruke scss syntax fordi du ikke er vant til å jobbe med det verktøyet. Jeg kan iaf. lenke til dokumentasjon om ampersand selector her:
https://sass-lang.com/documentation/style-rules/parent-selector/

Reply:
Jeg har aldri sagt at du ikke skal kunne bruke scss-syntaks. Poenget mitt er at det ofte blir unødvendig kognitiv last å måtte manuelt legge sammen strings for å se hvilket klassenavn det er snakk om. Syns parent-selector er helt fint å bruke når man bruker det til f.eks. pseudo-selectors som i eksempelet du lenker til, men om man bare bruker det for å legge på et suffiks (med unntak av om man f.eks. bruker BEM) syns jeg det ofte er unødvendig. Ser iaf ingen positive sider ved å bruke det, bare negative sider (kognitiv last)

Reply:
Ref møtet vi hadde om CSS er det en variant av BEM vi bruker med konkatenating av module for Element, og med Modifier. I den sammenheng er det særlig nyttig å kunne konkatenere klassenavnene. Det er heller ikke slik at jeg har innført det nylig, men det er noe som er konsekvent i hele kodebasen og som vi har hatt et møte om.